packages/react/src/menu/popup/MenuPopup.tsx

```diff
@@ -92,6 +93,7 @@ export const MenuPopup = React.forwardRef(function MenuPopup(
       popupProps,
       transitionStatus === 'starting' ? DISABLED_TRANSITIONS_STYLE : EMPTY_OBJECT,
       elementProps,
+      { 'data-rootownerid': rootId } as Record<string, string>, // This should be replaced with the actual root owner ID if available
```

Comment:
I'm struggling to understand  this comment

Also we've been using `{ ['data-rootownerid' as string]: rootId }`

Reply:
I forgot to remove the comment.

As for `{ ['data-rootownerid' as string]: rootId }` - this makes the property name computed, which is unnecessary. `{ 'data-rootownerid': rootId } as Record<string, string>` does not affect runtime code at all.